### DISCUSSION TITLE
Build opentelemetry-instrument-java from source fail

### DISCUSSION THREAD

**Discussion by lmssy** [open-telemetry/opentelemetry-java-instrumentation#3248] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/17894328/121641166-6c55e700-cac1-11eb-8c46-54b3a370b596.png)

![image](https://user-images.githubusercontent.com/17894328/121641396-b6d76380-cac1-11eb-9444-8b2b499e514c.png)

I have clone the source and try to build by gradle（without changing anything）, but it reports above error.

Need support, thx in advance!

**Comment by anuraaga** [comment, marked as answer]
Hi @lmssy - please use `./gradlew` (the wrapper in our repository) rather than `gradle` to ensure you are using a Gradle version compatible with our build.

**Reply by anuraaga** [reply]
Oops - nevermind I see there is a `w`. Can you run with `--info` for more logging?

**Reply by anuraaga** [reply]
Note the version is defined here

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/settings.gradle#L9

Did `settings.gradle` somehow get lost from your folder?

**Reply by lmssy** [reply]
Thanks a lot, I pull the latest code now it's ok
